src/platform/testing/unit/mocha-setup.js

```diff
@@ -30,8 +36,21 @@ export default function setupJSDom() {
 
   // Prevent warnings from displaying
   /* eslint-disable no-console */
-  console.error = () => {};
-  console.warn = () => {};
+  if (process.env.LOG_LEVEL === 'debug') {
+    console.error = (error, reactError) => {
+      if (reactError instanceof Error) {
+        console.log(filterStackTrace(reactError.stack));
+      } else if (error instanceof Response) {
```

Comment:
The api helper (incorrectly) returns a Response object when rejecting, this formats it so we don't see a weird printed Response object.